python/nnvm/frontend/keras.py

```diff
@@ -349,7 +349,14 @@ def _convert_concat(insym, keras_layer, _):
 
 
 def _convert_reshape(insym, keras_layer, _):
-    return _sym.reshape(insym, keras_layer.shape)
+    shape = keras_layer.shape if hasattr(keras_layer, 'shape') else \
```

Comment:
is it clearer to write the code as:
if hasattr(keras_layer, 'shape'):
   ...
elif hasattr(keras_layer, 'target_shape'):
   ...
else:
   raise TypeError("message")

Reply:
Ok, just ignore the last comment